modules/book_counter.js

```diff
@@ -0,0 +1,19 @@
+// Show a counter next to link1 after adding a book
+export function showBookCounter(
+  oldLength, newLegth, bookCounter,
+  allRecent = document.querySelectorAll('.recent-added'),
+) {
+  bookCounter.classList.remove('highlight-counter');
+  if (newLegth > oldLength) {
+    bookCounter.innerHTML = allRecent.length;
+    bookCounter.style.display = 'initial';
+    bookCounter.classList.add('highlight-counter');
+  }
+}
+
+// Remove counter when clicking link1
+export function hideBookCounter(navLinks, bookCounter, hash = window.location.hash) {
```

Comment:
- [ ] It would be better to use arrow functions here too, as this is a better practice in ES6, please do this for all the functions.